pandas/_libs/tslibs/timezones.pyx

```diff
@@ -72,20 +68,31 @@ cdef bint is_utc_zoneinfo(tzinfo tz):
     return tz is utc_zoneinfo
 
 
+cdef bint is_utc_pytz(tzinfo tz):
+    if tz is None:
+        return False
+
+    global utc_pytz
```

Comment:
Does this have any performance impacts? Referencing a global from a cdef function like this feels like it would really slow things down

Reply:
Yeah I was copying this pattern from zoneinfo above but I don't think we need this since `pytz.UTC` is just a singleton that's always there